embedded/store/immustore.go

```diff
@@ -565,11 +572,9 @@ func OpenWith(path string, vLogs []appendable.Appendable, txLog, cLog appendable
 	}
 
 	if store.synced {
-		store.cLogBuf = make([]byte, cLogEntrySize*opts.MaxActiveTransactions)
-
 		go func() {
 			for {
-				committedTxID := store.lastCommittedTxID()
+				committedTxID := store.LastCommittedTxID()
```

Comment:
If an error occurs (and if syncFrequency is not set), this runs into an infinite loop

Reply:
I am assuming this happens because sync frequency is not set on defaultdb